task/control-plane-agent/src/mgs_common.rs

```diff
@@ -405,6 +406,72 @@ impl MgsCommon {
             Err(e) => Err(e.into()),
         }
     }
+
+    #[cfg(not(feature = "vpd"))]
+    pub(crate) fn vpd_lock_status_all(
+        &self,
+        _buf: &mut [u8],
+    ) -> Result<usize, SpError> {
+        Err(SpError::Vpd(GatewayVpdError::NotImplemented))
+    }
+
+    #[cfg(feature = "vpd")]
+    pub(crate) fn vpd_lock_status_all(
+        &self,
+        buf: &mut [u8],
+    ) -> Result<usize, SpError> {
```

Comment:
Commit [4c3681dff78764374d6d873db5a4d2be272009f4](https://github.com/oxidecomputer/hubris/commit/cb27900aae24a90c2a162a0b42eb13a59af6f197) added event counters for each MGS message based on the ringbuf entries recorded for those messages. Can we add a variant for `VpdLockStatusAll` to `MgsMessage`, and record it in the ringbuf here, as well?

Reply:
lovely, thanks!